asserts/sysdb/sysdb_test.go

```diff
@@ -64,7 +65,16 @@ func (sdbs *sysDBSuite) SetUpTest(c *C) {
 
 	sdbs.extraTrusted = []asserts.Assertion{trustedAcct, trustedAccKey}
 
+	otherAcct := assertstest.NewAccount(signingDB, "gener1c", map[string]interface{}{
+		"account-id": "gener1c",
+		"validation": "certified",
+		"timestamp":  "2015-11-20T15:04:00Z",
```

Comment:
I think this timestamp string is repeated 3 times in this func now. Maybe it's time to define a var, but I really don't care much.